molr-commons/src/main/java/io/molr/commons/domain/BlockAttribute.java

```diff
@@ -1,5 +1,11 @@
 package io.molr.commons.domain;
 
 public enum BlockAttribute {
-	BREAK, IGNORE, ON_ERROR_SKIP_SEQUENTIAL_SIBLINGS
+	BREAK,
+	IGNORE,
+	ON_ERROR_SKIP_SEQUENTIAL_SIBLINGS,
+	/*
+	 * If execution strategy is PROCEED_ON_ERROR blocks marked with attribute will still cause strand execution being abort
+	 */
+	ON_ERROR_FORCE_QUIT
```

Comment:
Not fully shure about the name ... 'quit' is a term we never used up to now...
Would
```
ON_ERROR_FAIL_MISSION
```
be more accurate?